docs/adr/0016-automatic-aggregate-creation.md

```diff
@@ -41,7 +41,7 @@ implement their domain logic in a clear and concise way.
 From the engine implementors perspective it seems that a non-trivial amount of
 validation logic needs to be implemented within the scope to verify that the
 application code is using the scope correctly, even though the engine does not
-really benefit from this requirements.
+really benefit from these requirements.
```

Comment:
Changes to this file seem like a bit of a drive-by...

Reply:
Good call. I originally copied these ADRs to create this one and fixed the same typo in both places, but it only exists here now.

Reply:
I've applied these changes to `main` separately.